cmd/vpn_server/app/app.go

```diff
@@ -67,6 +70,40 @@ func run(_ context.Context, log logr.Logger) error {
 		return err
 	}
 
+	if !cfg.IsHA {
+		log.Info("setting up iptables rules for Envoy")
+		ipTable, err := network.NewIPTables(log, iptables.ProtocolIPv4)
+		if err != nil {
+			return err
+		}
+
+		for _, nw := range cfg.PodNetworks {
+			if nw.IsIPv4() {
+				err = ipTable.AppendUnique("nat", "OUTPUT", "-m", "owner", "--uid-owner", "65532", "-d", nw.String(), "-j", "NETMAP", "--to", constants.ShootPodNetworkMapped)
+				if err != nil {
+					return err
+				}
+			}
+		}
+		for _, nw := range cfg.ServiceNetworks {
+			if nw.IsIPv4() {
+				err = ipTable.AppendUnique("nat", "OUTPUT", "-m", "owner", "--uid-owner", "65532", "-d", nw.String(), "-j", "NETMAP", "--to", constants.ShootSvcNetworkMapped)
+				if err != nil {
+					return err
+				}
+			}
+		}
+		for _, nw := range cfg.NodeNetworks {
+			if nw.IsIPv4() {
+				err = ipTable.AppendUnique("nat", "OUTPUT", "-m", "owner", "--uid-owner", "65532", "-d", nw.String(), "-j", "NETMAP", "--to", constants.ShootNodeNetworkMapped)
+				if err != nil {
+					return err
+				}
+			}
+		}
```

Comment:
This code assumes that there is only ever at most one IPv4 range in the slices. Should we handle more than one or at least detect the case and panic/handle it in some deterministic way?